File: semestr-8/Programowanie obiektowe w Java/alexander/POJ/Archiwum/2022s/IZ08IO3/Commands/src/Sportsmen.java
public class Sportsmen
{
   private String surname;
   public Sportsmen(String surname) { this.surname=surname; }
   public String getSurname() { return surname; }

   public String jump() { return "Jumping"; }
   public String run() { return "Running"; }
   public String swim() { return "Swimming"; }
   String execute(SportsmenCommand cmd) { return surname+": "+cmd.execute(this); }
}

File: semestr-8/Jezyk C#/alexander/Csharp/Archiwum/2022s/ID06PD1/StrategyWkitoria/Strategy/Sportsmen.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Strategy
{
    internal class Sportsmen
    {
        public string surname { get; private set; }
        public Sportsmen(string surname) { this.surname = surname; }
        public string Jump() { return "Jumping"; }
        public string Run() { return "Running"; }
        public string Swim() { return "Swimming"; }
        public string execute(SportsmenCommand cmd) { return surname + ": " + cmd.execute(this); }
    }
}
